samples/nrf9160/mqtt_simple/overlay-tls.conf

```diff
@@ -0,0 +1,8 @@
+# To build the mqtt simple sample with TLS support
+# add the following to your west build command:
+# -- -DOVERLAY_CONFIG=overlay-tls.conf
+
+# MQTT TLS
+CONFIG_MQTT_LIB_TLS=y
+CONFIG_MQTT_BROKER_HOSTNAME="test.mosquitto.org"
```

Comment:
I tested that it's also possible to connect to the default broker `mqtt.eclipse.org`

Reply:
Very good, without changing the CA?

Reply:
Changed the default broker to mqtt.eclipse.org and added the corresponding CA.